articles/communication-services/concepts/email/email-optout-management.md

```diff
@@ -15,6 +15,9 @@ ms.service: azure-communication-services
 
 [!INCLUDE [Public Preview Notice](../../includes/public-preview-include-document.md)]
 
+> [!IMPORTANT]
+> The functionality described in this article is currently only available in SDK version [Azure.ResourceManager.Communication/1.2.0-beta.4](https://www.nuget.org/packages/Azure.ResourceManager.Communication/1.2.0-beta.4).
+  
```

Comment:
Let's change this to latest pre-release or beta version instead of a specific one. Otherwise, once a new version of the SDK is released, this will be obsolete.